jmix-ui/ui/src/main/java/io/jmix/ui/component/impl/FilterImpl.java

```diff
@@ -208,6 +207,35 @@ public void setAutoApply(boolean autoApply) {
         }
     }
 
+    @Nullable
+    @Override
+    public KeyCombination getShortcutCombination() {
+        return searchButton.getShortcutCombination();
+    }
+
+    @Override
+    public void setApplyShortcutCombination(@Nullable KeyCombination applyShortcut) {
+        KeyCombination oldValue = this.applyShortcut;
+        if (!Objects.equals(oldValue, applyShortcut)) {
+            this.applyShortcut = applyShortcut;
+
+            if (applyShortcut != null) {
+                searchButton.setShortcutCombination(applyShortcut);
+            } else {
+                searchButton.setShortcut(null);
+            }
+        }
+    }
+
+    @Override
+    public void setApplyShortcut(@Nullable String applyShortcut) {
+        if (applyShortcut != null) {

```

Comment:
Use ternary operator

Reply:
`If` operator is useless anyway
KeyCombination.create(applyShortcut) accepts `@Nullable` values